support/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+exclude = .git,.tox,__pycache__,.eggs,dist,.venv*,docs,build
```

Comment:
Pre-commit is responsible for running this; we don't need the exclusions listed here, I think.

Reply:
I can remove it from the repo... or I can move it to the root and add flake8 to the pre-commit config? I had flake8 for my own workflow while working on the script, which is how this snuck in here.

Reply:
Let's add flake8 to the pre-commit config and move this to the repo root! :partying_face: